pkg/report/github/github.go

```diff
@@ -105,8 +105,17 @@ func (w Writer) Write(ctx context.Context, report types.Report) error {
 		manifest.Name = string(result.Type)
 		// show path for language-specific packages only
 		if result.Class == types.ClassLangPkg {
-			manifest.File = &File{
-				SrcLocation: result.Target,
+			if report.ArtifactType == "container_image" {
+				// Replacing `source_location` by the image name and tag
+				image_reference := strings.Join(report.Metadata.RepoTags, ", ")
```

Comment:
If we used `report.ArtifactName` instead we would miss the image tag.

Reply:
Could we use "RepoDigests" instead, as that included both the tag and digest? That would address the comment I just put on the PR discussion

Reply:
Just to confirm using `RepoDigests` would work I did some testing and it seems to be working fine.

Each time you scan your image with a new sha it will close all the previously open alerts for that image and only keep the latest version open. 
But you're still able to see the previously closed alerts by simply using the `is:closed` filter and searching for a specific manifest works fine too.

Reply:
As suggested by https://github.com/aquasecurity/trivy/pull/5999#discussion_r1469105434

We could store all possible information using `<image_registry>/<image_name>:<tag>@sha256:<image_hash> - <filePath>`
![Screenshot_20240202_094700](https://github.com/aquasecurity/trivy/assets/72691393/a2de2962-39a2-408b-8aa1-58f299d7a449)

I tested it and although it's very long, filtering seems to work too (the `image_hash` is fake).

Reply:
Yeah, showing the package location (while having those disadvantages) will help with investigations about what actually needs to be fixed/updated